src/borg/archiver/debug_cmd.py

```diff
@@ -265,6 +266,61 @@ def do_debug_id_hash(self, args, repository, manifest):
         print(id.hex())
         return EXIT_SUCCESS
 
+    @with_repository(compatibility=Manifest.NO_OPERATION_CHECK)
+    def do_debug_parse_obj(self, args, repository, manifest):
+        """parse object file into meta dict and data (decrypting, decompressing)"""
+
+        # get the object from id
+        hex_id = args.id
+        try:
+            id = unhexlify(hex_id)
+            if len(id) != 32:  # 256bit
+                raise ValueError("id must be 256bits or 64 hex digits")
+        except ValueError as err:
+            print("object id %s is invalid [%s]." % (hex_id, str(err)))
+            return EXIT_ERROR
+
+        with open(args.object_path, "rb") as f:
+            object_file = f.read()
```

Comment:
you need to be more careful with names. names are important - if they are appropriate, it makes the code much easier to read and understand.

if something is called `..._file` the first guess would be it is a python file object. a slightly sloppier guess might be that it is a OS-level file handle or a file name or path.

but, what you have in there is just a bytes object (binary data), so `data` or `cdata` (considering 287) or `data_encrypted` (see 318) would be more appropriate.